searchlib/src/vespa/searchlib/docstore/filechunk.cpp

```diff
@@ -250,8 +250,8 @@ void
 FileChunk::remove(uint32_t lid, uint32_t size)
 {
      (void) lid;
-     _erasedCount++;
-     _erasedBytes += adjustSize(size);
+     _erasedCount.store(getBloatCount() + 1, std::memory_order_relaxed);
```

Comment:
At first sight it's a bit confusing that `_erasedCount` is updated with the result of `getBloatCount()` (even though that just relaxed-loads `_erasedCount`). Should the function (or variable) be renamed so that they match?

Reply:
Yes, they should. Fixed